src/auth/auth0ProviderWithHistory.js

```diff
@@ -18,6 +18,7 @@ const Auth0ProviderWithHistory = ({ children }) => {
       clientId={clientId}
       redirectUri={window.location.origin}
       onRedirectCallback={onRedirectCallback}
+      cacheLocation="localstorage"
```

Comment:
One letter yesterday, one line today! This makes sense though, that it was clearing that cache every time and that was why it was constantly trying to log in again because the token wasn't there.